fbw-a32nx/src/systems/instruments/src/SD/Pages/Apu/Apu.tsx

```diff
@@ -171,9 +172,9 @@ const ApuBleed = ({ x, y } : ComponentPositionProps) => {
                 <text
                     x={44}
                     y={48}
-                    className={`FontLarger Right ${adir1ModeSelectorKnob === 1 ? 'Green' : 'Amber'}`}
+                    className={`FontLarger Right $className= ${apuBleedPressAvailable ? 'Green' : 'Amber'}`}
                 >
-                    {adir1ModeSelectorKnob === 1 ? displayedBleedPressure : 'XX'}
+                    {apuN.isNormalOperation() && adir1ModeSelectorKnob === 1 ? displayedBleedPressure : 'XX'}
```

Comment:
To avoid bad surprises.

```suggestion
                    {(apuN.isNormalOperation() && adir1ModeSelectorKnob === 1) ? displayedBleedPressure : 'XX'}
```